tools/rna_tools/targetfinder/TargetFinder.xml

```diff
@@ -0,0 +1,66 @@
+<tool id="targetfinder" name="targetfinder" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="1.07">targetfinder</requirement>
+        <requirement type="package" version="35.4.12">fasta</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+
+    <command><![CDATA[
+        targetfinder.pl
+            -s "$s"
+            -d $d
+            -q "$q"
+            -c $c
+            -t $t
+            -p "$p"
+            $r
+            > $output1
+    ]]></command>
+    <inputs>
+        <param label="Small RNA sequence (RNA or DNA, 5'->3')" help="(-s)" name="s" type="text"/>
```

Comment:
If your `name` is the same as your `--argument` you can use <param argument="-s">.
name is in this case not needed and the help "(-s)" will be created automatically.